cmake/test/CMakeLists.txt

```diff
@@ -61,13 +55,17 @@ if(TEST_SHARED)
         )
     endif()
 
+    # TODO: DG: I'm not sure what this is for exactly, and now the SDL3::SDL3_main target doesn't
+    #  exist anymore (because SDL3_main is a header-only lib instead of static lib) so I'm not sure
+    #  if this is still needed
     # SDL3_LIBRARIES does not support creating a cli SDL3 application
     # (it is possible that SDL3_main is a stub, but we don't know for sure)
-    if(NOT TARGET SDL3::SDL3_main)
+    #if(NOT TARGET SDL3::SDL3_main)
         add_executable(cli-shared-vars main_cli.c)
         target_link_libraries(cli-shared-vars PRIVATE ${SDL3_LIBRARIES})
         target_include_directories(cli-shared-vars PRIVATE ${SDL3_INCLUDE_DIRS})
-    endif()
+    #endif()
```

Comment:
This tested whether a cli application can be built with the `SDL3_*` cmake variables.
The `SDL3_*` variables expect to build a GUI application (with `WinMain`), which we want to avoid here.

So you can safely remove the `if` and test this case unconditionally.

Reply:
Thanks for the comment!
So should this be something like `if(WIN32)` instead of `if(NOT TARGET SDL3::SDL3_main)` now? This is only relevant for desktop Windows, right?

Reply:
No the test can be run anyways.
The goal of `cmake/test/CMakeLists.txt` is to test whether a user can create a cli/gui/library by using the `SDL_*` variables or the `SDL3::*` targets, using a shared/static SDL library.
Since the variables are now "safe" to use for cli, all tests can be run.

Reply:
ok, I'll remove the (already commented out) ifs while keeping cli-*-vars - thanks! :)